tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -344,19 +345,39 @@ public void setCachingEnabled(boolean enabled) {
                 }
             };
 
-    // map relativeTimePattern counts to numeric examples
-    public static final Map<String, String> COUNTS =
+    // map relativeTimePattern counts to possible numeric examples.
+    // For few , many, and other there is not a single number that is in that category for
+    // all locales, so we provide a list of values that might be good examples and use the
+    // first that is in the category for the locale. Decimal fractions should be at the end.
+    public static final Map<String, List<String>> COUNTS =
             new HashMap<>() {
                 {
-                    put("zero", "0");
-                    put("one", "1");
-                    put("two", "2");
-                    put("few", "3");
-                    put("many", "5");
-                    put("other", "10");
+                    put("zero", List.of("0"));
+                    put("one", List.of("1"));
+                    put("two", List.of("2"));
+                    put("few", List.of("3" /*gv*/, "20" /*gv*/));
+                    put(
+                            "many",
+                            List.of(
+                                    "11" /*ar pl ru uk*/,
+                                    "1000000" /*ca es fr it pt*/,
+                                    "6" /*cy*/,
+                                    "7" /*ga*/,
+                                    "21" /*kw*/,
+                                    "0.5" /*cs sk lt*/));
+                    put(
+                            "other",
+                            List.of(
+                                    "100" /*ar cs de en es fr he it*/,
+                                    "22" /*lv prg*/,
+                                    "23" /*gv*/,
+                                    "14" /*ceb fil tl*/,
+                                    "0.5" /*pl ru uk*/));
```

Comment:
Couldn't this call `Map<Count, Set<Double>> SupplementalDataInfo.PluralInfo.getCountToExamplesMap()` ?

Reply:
But then we still need to pull one example from the set, and what I was trying to do here was find examples that would contrast in an obvious way with the examples for other plural forms (and that could also be used across multiple locales), instead of e.g. just pulling the first example from the set returned by `SupplementalDataInfo.PluralInfo.getCountToExamplesMap`. Anyway I can explore other options in a future PR, this PR at least gets correct behavior in for the vetters, even if the code is not as clean as it could be.